mnefun/_mnefun.py

```diff
@@ -610,21 +610,27 @@ def fetch_raw_files(p, subjects, run_indices):
         raw_dir = op.join(subj_dir, p.raw_dir)
         if not op.isdir(raw_dir):
             os.mkdir(raw_dir)
-        finder_stem = 'find %s ' % p.acq_dir
-        # build remote raw file finder
-        fnames = get_raw_fnames(p, subj, 'raw', True, False, run_indices[si])
+        fnames = get_raw_fnames(p, subj, 'raw', True, False,
+                                run_indices[si])
         assert len(fnames) > 0
-        finder = (finder_stem +
-                  ' -o '.join(['-type f -regex ' + _regex_convert(f)
-                               for f in fnames]))
+        # build remote raw file finder
+        if isinstance(p.acq_dir, string_types):
+            use_dir = [p.acq_dir]
+        else:
+            use_dir = p.acq_dir
```

Comment:
yes exactly, the idea is just to ensure it's a list `list` to simplify subsequent operations